pkg/clusters/utils.go

```diff
@@ -75,3 +77,36 @@ func GetIngressLoadbalancerStatus(ctx context.Context, c Cluster, namespace stri
 		return nil, fmt.Errorf("%T is not a supported ingress type", ingress)
 	}
 }
+
+// CreateNamespace create customized namespace
+func CreateNamespace(ctx context.Context, cluster Cluster, namespace string) error {
+	nsName := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+	attempts := 5
+	for attempts > 0 {
```

Comment:
non-blocker (question): is the use of retries here intended to try and accomodate for failures connecting to, or in making requests of the Kubernetes API?

Reply:
Retry for K8S namespace creation failure.